content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -23,21 +23,26 @@ a building block.  kops builds on the kubeadm work.
 
 {{% /capture %}}
 
-{{% capture body %}}
+{{% capture prerequisites %}}
 
-## Creating a cluster
+* You must have [kubectl](/docs/tasks/tools/install-kubectl/) installed in order for kops to work.
```

Comment:
Do you need, `in order for kops to work`?

Reply:
I have removed this.